OpenRA.Mods.Common/ActorInitializer.cs

```diff
@@ -24,6 +24,16 @@ public class FacingInit : IActorInit<int>
 		public int Value(World world) { return value; }
 	}
 
+	public class DelayInit : IActorInit<int>
```

Comment:
`DelayInit` is too generic and may conflict with other Inits in the future (also don't forget that these are automatically bridged to `map.yaml`). Please rename this to `MoveIntoWorldDelayInit` to make its use clear, even if you don't want to explicitly gate the activity on it existing.